### DISCUSSION TITLE
enter to go to menu nothing pops up

### DISCUSSION THREAD

**Discussion by Brendanbam** [itsToggle/plex_debrid#104] [Q&A] [answered]
after completing the setup through python i hit enter to load the menu and nothing shows up

**Comment by itsToggle** [comment, marked as answer]
Hey, I thought this was fixed but are you by any chance on windows using the python console? Try running the script from a terminal (cmd)

**Reply by Brendanbam** [reply]
__                  __     __         _     __
    ____  / /__  _  __    ____/ /__  / /_  _____(_)___/ /
   / __ \/ / _ \| |/_/   / __  / _ \/ __ \/ ___/ / __  /
  / /_/ / /  __/>  <    / /_/ /  __/ /_/ / /  / / /_/ /
 / .___/_/\___/_/|_|____\__,_/\___/_.___/_/  /_/\__,_/
/_/               /_____/

Done!

Press Enter to continue to the main menu:
Traceback (most recent call last):
  File "C:\Users\user\Downloads\plex_debrid-main\plex_rd.py", line 3815, in <module>
    ui.run()
  File "C:\Users\user\Downloads\plex_debrid-main\plex_rd.py", line 3788, in run
    if ui.setup():
  File "C:\Users\user\Downloads\plex_debrid-main\plex_rd.py", line 3742, in setup
    ui.save()
  File "C:\Users\user\Downloads\plex_debrid-main\plex_rd.py", line 3749, in save
    with open('settings.json', 'w') as f:
PermissionError: [Errno 13] Permission denied: 'settings.json'

[process exited with code 1 (0x00000001)]

i get this

**Reply by Brendanbam** [reply]
C:\Users\user>python "C:\Users\user\Downloads\plex_debrid-main\plex_rd.py"
'python' is not recognized as an internal or external command,
operable program or batch file.

**Reply by itsToggle** [reply]
First error tells you that the plex_debrid script does not have permission to save your settings as a file. Open the cmd with admin privilege?

Second error tells you that python is not in your PATH variable. Add python to your PATH variable :)

You can always just google these errors, the first hit is usually the solution.

**Reply by Brendanbam** [reply]
everything is solved i had to use py instead of python
